docs/usage.rst

```diff
@@ -133,6 +134,56 @@ would rather see the equivalent XRootD endpoints for the files:
 
 The data files can be downloaded via XRootD protocol using the **xrdcp** command.
 
+List directory
+--------------
+In order to get a list of files belonging to a EOSPUBLIC path, please use
+the **list-directory** command:
+
+It uses XRootD protocol to list the data files.
+
+.. code-block:: console
+
+    $ cernopendata-client list-directory /eos/opendata/cms/validated-runs/Commissioning10
+    Commissioning10-May19ReReco_7TeV.json
+    Commissioning10-May19ReReco_900GeV.json
+
+**Iterate recursively**
+
+Note that you can use ``--recursive`` command-line option if you
+would rather want to iterate through subdirectories:
+
+.. code-block:: console
+
+    $ cernopendata-client list-directory /eos/opendata/cms/validated-runs --recursive
+    Commissioning10-May19ReReco_7TeV.json
+    Commissioning10-May19ReReco_900GeV.json
+    Cert_190456-208686_8TeV_22Jan2013ReReco_Collisions12_JSON.txt
+    Cert_160404-180252_7TeV_ReRecoNov08_Collisions11_JSON.txt
+    Cert_136033-149442_7TeV_Apr21ReReco_Collisions10_JSON_v2.txt
+
+**Iterate recursively with timeout**
+
+Note that you can use ``--recursive --timeout <time_in_seconds>`` command-line option if you
```

Comment:
If you would like to list a directory that contains a large amount of files, you can specify `--timeout` option in order to exit after a certain amount of time.  The default timeout is 60 seconds.